testnet-contracts/contracts/Processor.sol

```diff
@@ -1,7 +1,7 @@
 pragma solidity ^0.5.0;
```

Comment:
should be updated to latest solidity

Reply:
Can do this in a future PR, let's get the contract tooling integrated first. Will likely address this when adding in the contract validator set.